journal.py

```diff
@@ -45,6 +50,51 @@ def init_db():
         db.commit()
 
 
+@subscriber(NewRequest)
+def open_connection(event):
+    request = event.request
+    settings = request.registry.settings
+    request.db = connect_db(settings)
+    request.add_finished_callback(close_connection)
+
+
+def close_connection(request):
+    """close the database connection for this request
+
+    If there has been an error in the processing of the request, abort any
+    open transactions.
+    """
+    db = getattr(request, 'db', None)
+    if db is not None:
+        if request.exception is not None:
+            db.rollback()
+        else:
+            db.commit()
+        request.db.close()
+
+
+def write_entry(request):
+    title = request.params.get('title', None)
+    text = request.params.get('text', None)
+    created = datetime.datetime.utcnow()
+    request.db.cursor().execute(INSERT_ENTRY, [title, text, created])
+
+
+# @view_config(route_name='home', renderer='string')
```

Comment:
better to delete than to comment out.  trust version control to keep old code for reference.